openrtx/src/ui/default/ui.c

```diff
@@ -1900,6 +1929,41 @@ void ui_updateFSM(bool *sync_rtx)
                     _ui_menuBack(MENU_TOP);
                 break;
 #endif
+#ifdef CONFIG_SPECTRUM
+            // Spectrum menu screen
+            case MENU_SPECTRUM:
+            // Go to the Spectrum Settings if we press ENTER
+                if(msg.keys & KEY_ENTER)
+                {
+                    display_init();
```

Comment:
Here you are re-initializing the display driver, are you sure that is what you really want to do?